Game/Assets/Scripts/Timer.cpp

```diff
@@ -0,0 +1,20 @@
+﻿#include "Timer.h"
+#include "Score.h"
+
+namespace Engine
+{
+	void Timer::onUpdate(const Engine::DeltaTime& dt)
+	{
+		time += dt.getSeconds();
+		//GE_CORE_INFO("Timer: {0}", time);
+		if (doesEntityExist("score"))
+		{
+			//GE_CORE_INFO("Score: {0}", static_cast<Score*>(getEntity("score").getComponent<ScriptComponent>().m_instance)->getScore());
```

Comment:
can we replace with a TODO and a subsequent trello ticket

Reply:
No need, this is old and has been removed